addons/image-uploader/userscript.js

```diff
@@ -136,7 +136,12 @@ export default async function ({ addon, msg, console }) {
       const data = await res.json();
 
       if (data.status === "ok") {
-        insert(textBox, `[img]https://assets.scratch.mit.edu/get_image/.%2E/${data["content-name"]}[/img]`);
+        insert(
+          textBox,
+          (await addon.self.getEnabledAddons()).includes("markdown-forums")
+            ? `![](https://assets.scratch.mit.edu/get_image/.%2E/${data["content-name"]})`
+            : `[img]https://assets.scratch.mit.edu/get_image/.%2E/${data["content-name"]}[/img]`
```

Comment:
I think this looks perfectly OK, but do we want to allow dynamic enable & disable of the new `markdown-forums` addon? Won't it be a mess to revert these syntax changes back?

Reply:
It currently allows dynamic enable/disable without converting anything back and forth. This is probably safer than trying to accurately convert between BBCode and Markdown.

Reply:
Yeah that sounds like a mess